<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EndusersClusterMngmtTabs</class>
 <widget class="QDialog" name="EndusersClusterMngmtTabs">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XIPE - Endusers Cluster Management</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelFindUsers">
         <property name="text">
          <string>Find Users Where:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxType"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxParameter"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditQueryData"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnFindUsers">
         <property name="text">
          <string>Find Users</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 170, 255);</string>
         </property>
         <property name="text">
          <string>Enduser is in Cluster</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string>Enduser NOT in Cluster</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTabWidget" name="tabWidgetClusters">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="elideMode">
          <enum>Qt::ElideNone</enum>
         </property>
         <property name="tabsClosable">
          <bool>false</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
         <property name="tabBarAutoHide">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="btnRemoveEndusersFromCluster">
         <property name="text">
          <string>Remove selected Endusers from Cluster</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAddEndusersToCluster">
         <property name="text">
          <string>Add Selected Endusers to Cluster</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="labelEndusersInfoMessage">
          <property name="text">
           <string>To begin, use the Query Box at the top to execute a query to find specific users!</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
